src/main/kotlin/no/nav/familie/ef/iverksett/infrastruktur/IverksettController.kt

```diff
@@ -17,8 +18,8 @@ import org.springframework.web.bind.annotation.RestController
 class IverksettController(val iverksettService: IverksettService) {
 
     @PostMapping(path = ["/iverksett"])
-    fun iverksett(@RequestBody vedtakJSON: IverksettJson): ResponseEntity<String> {
-        return ResponseEntity("Iverksett", HttpStatus.OK)
+    fun iverksettdummy(@RequestBody iverksettJson: IverksettJson): ResponseEntity<IverksettJson> {
```

Comment:
I andre appene i teamfamilie navngis det Dto i stedet for Json, men det går fint for min del å bruke Json som postfix. Mulig vi må ta hensyn til det.